### ISSUE TITLE
Running `eslint` with version 12.0.0 of `@nuxtjs/eslint-config` turns on the `--report-unused-disable-directive` flag

### ISSUE THREAD

**Issue by ericpromislow** [nuxt/eslint#243]
Easiiest way to repro this is to pull down

https://github.com/rancher-sandbox/rancher-desktop

You need to first run `npm i` (goes much faster if you change the `"postinstall"` line in `package.json` around line 41 to `echo skipping node...`).

Then run a command like:

```
node node_modules/eslint/bin/eslint.js --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue  pkg/rancher-desktop/backend/k3sHelper.ts
```

For this file, I get this output:
```
/Users/ericp/workspace/rancher/desktop/pkg/rancher-desktop/backend/k3sHelper.ts
  47:3  warning  Unused eslint-disable directive (no problems were reported from 'camelcase')
  50:5  warning  Unused eslint-disable directive (no problems were reported from 'camelcase')

✖ 2 problems (0 errors, 2 warnings)
  0 errors and 2 warnings potentially fixable with the `--fix` option.
```

Now if the version field in `package.json` is `"^12.0.0"` (shouldn't be much higher if this is still open), change  it to `"11.0.0"`, rerun `npm i` and the above `node ... eslint.js` , and you should get no output.

I looked at the diffs for tags v11.0.0 and v12.0.0 and didn't see any code that looks like it's causing a problem, but I didn't trace into differences in the dependencies.

**Comment by antfu** [comment]
This issue was created for the legacy configs for Nuxt 2, which have been deprecated for a while and removed recently in #534. The issue is likely no longer relevant to the latest version. If you still have a similar issue with the latest config, please open a new issue with minimal reproduction. Thanks.
